packages/editor-ui/src/plugins/i18n/docs/ADDENDUM.md

```diff
@@ -2,27 +2,33 @@
 
 ## Base text
```

Comment:
thanks for updating the guide. can you also update section that says we should use first words of a message? I think I saw this in a previous PR. Instead names should be more descriptive.

Reply:
That section was in [the other PR](https://github.com/n8n-io/n8n/pull/2693/files) that was not merged. I intentionally left that part out.